docs/imagecustomizer/how-to/quick-start.md

```diff
@@ -35,26 +35,52 @@ nav_order: 1
         xfsprogs zstd cryptsetup-bin grub2-common binutils lsof
      ```
 
-   - For Azure Linux 2.0, run:
+   - For Azure Linux 2.0 on x86_64, run:
```

Comment:
I think this might be getting a little spammy. Would it make sense to provide a base set of instructions for Azure Linux, and then provide additional instructions for both x86_64 and AZL3?